### ISSUE TITLE
Various problems with the new docker image

### ISSUE THREAD

**Issue by gissehel** [droppyjs/droppy#14]
There are various problems with the new Dockerfile image.
* Script `docker-start.sh` use `sudo` while the sudo package is not installed
* Script `docker-start.sh` use `sudo` under the name `/bin/sudo` while when installed, the sudo package provide sudo under the name `/usr/bin/sudo`
* The `Dockerfile` clean some files inside the image. But the layered structure of Docker images make it useless : the files are still present inside the image, they are just not accessible.

# Description of problem with layers

Inside the `Dockerfile`, we can find

```Dockerfile
RUN apt-get -y update
RUN apt-get -y install aria2 gnupg software-properties-common \
                       python3 git curl bash openssl
```

and latter

```Dockerfile
## Lets slim the image down!

# systemd uses 26.6 MB of space!
RUN apt-get -y remove --purge --auto-remove systemd

# remove our cache from apt-get
RUN rm -rf /var/cache/apt/archives/

# apt-get update cache (17 MB)
RUN rm -rf /var/lib/apt/lists/

# man page ~6 MB
RUN rm -rf /usr/share/man/

# unused locale data ~31 MB
RUN rm -rf /usr/share/locale/

# unused mandocs ~11 MB
RUN rm -rf /usr/share/doc/
```

Each time a new RUN instruction is present, a new layer is created in the image with all the file created or modified in this layer. If a subsequent layer remove files, the files won't be removed from the image, they will just be flagged as being "not accessible".

This is why a lot a `Dockerfile`s that try to minimize the size of an image use as few RUN instruction as possible, either by putting all instruction on the same line like:
```Dockerfile
RUN instuction1 && \
    instruction2 && \
    instruction3 && \
    instruction4 && \
    rm -rd /.../cache-folder1 && \
    rm -rd /.../cache-folder2
```
or by using a script
```Dockerfile
ADD install.sh /
RUN /bin/bash /install.sh && rm -f /install.sh
```

**Comment by markhughes** [comment]
Didn't realise that! Thanks for the information! I'll change that :)

**Comment by markhughes** [comment]
Resolved in #14 - thanks for the feedback! Please give some more if you have a chance!
